cli/src/cli-types.ts

```diff
@@ -1,15 +1,20 @@
 import { Type } from 'cmd-ts'
 import { existsSync } from 'node:fs'
 import { CliError, CliErrorCode } from './errors'
-import { loadFile } from './utils'
+import { isValidUrl, loadFile, loadJsonFromUrl } from './utils'
 
 export const ReadFile: Type<string, string> = {
-  async from(filePath) {
-    if (!existsSync(filePath)) {
-      throw new CliError(CliErrorCode.FileNotFound)
+  async from(source) {
+    if (await isValidUrl(source)) {
+      // load from Url
```

Comment:
I think we don't need this type of comments. Same as below.

If you want to document your code for reader, it could be nicer to write a short explanation above the function.